src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseController.java

```diff
@@ -211,4 +219,43 @@ public CaseController(
 
         return status(HttpStatus.CREATED).body(new CaseResource(caseDetails, content, ignoreWarning));
     }
+
+
+    @GetMapping(
+        path = "/cases/{caseId}/events",
+        headers = {
+            V2.EXPERIMENTAL_HEADER
+        },
+        produces = {
+            V2.MediaType.CASE_EVENTS
+        }
+    )
+    @ApiOperation(
+        value = "Retrieve an event by case and event IDs for dynamic display",
```

Comment:
Is this correct? Are we retrieving also by event IDS?

Reply:
Also I think we retrieve audit event and not a single event. Could we fix that too?

Reply:
This is also not for dynamic display. Could we say:
"Retrieve audit events by case ID"

Reply:
changed